.github/workflows/ci.yml

```diff
@@ -88,7 +88,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: fmt
-          args: --workspace -- --check
+          args: --all -- --check
```

Comment:
`--workspace` behaviour is actually the default, `--all` does something different (finds `path` dependencies and formats those, even outside the workspace), https://github.com/rust-lang/rustfmt/issues/3911

Reply:
Wow, that's confusing 😆